File: Module.Persist.TPM/Model/History/HistoricalBaseLine.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;
using System.ComponentModel.DataAnnotations;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(BaseLine))]
    public class HistoricalBaseLine : BaseHistoricalEntity<System.Guid>
    {
        public string ProductZREP { get; set; }
        public string DemandCode { get; set; }
        public DateTimeOffset? StartDate { get; set; }
        public double? InputBaselineQTY { get; set; }
        public double? SellInBaselineQTY { get; set; }
        public double? SellOutBaselineQTY { get; set; }
        public int? Type { get; set; }
        public DateTimeOffset? LastModifiedDate { get; set; }
    }
}

File: Module.Frontend.TPM/Client/model/tpm/baseline/HistoricalBaseLine.js
Ext.define('App.model.tpm.baseline.HistoricalBaseLine', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'BaseLine',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'ProductZREP', type: 'string', isDefault: true },
        { name: 'StartDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'DemandCode', type: 'string', hidden: false, isDefault: true }, 
        { name: 'InputBaselineQTY', type: 'float', hidden: false, isDefault: true },
        { name: 'SellInBaselineQTY', type: 'float', hidden: false, isDefault: true },
        { name: 'SellOutBaselineQTY', type: 'float', hidden: false, isDefault: true },
        { name: 'Type', type: 'int', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalBaseLines',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});
